proposals/lip-0051.md

```diff
@@ -2585,6 +2585,8 @@ def beforeCrossChainMessageForwarding(trs: Transaction, ccm: CCM, ccmFailed: boo
     if ccm.fee > 0:
         # This should never fail since it is checked in verifyCrossChainMessage.
         messageFeeTokenID = Interoperability.getMessageFeeTokenID(ccm.receivingChainID) # Always equal to getTokenIDLSK().
+        if messageFeeTokenID != getTokenIDLSK():
```

Comment:
I would remove the comment `# Always equal to getTokenIDLSK().` from the line above

Reply:
Done.